android/README.md

```diff
@@ -9,7 +9,20 @@
 
 Ping provides these Android samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.
 
-### Integrate with PingOne Advanced Identity Cloud / PingAM:
+The Ping SDK for Android enables you to integrate your SPA with our services.
+There are two methods of UI integration:
```

Comment:
With the big reorg, I think we should evolve away from these _two_ methods, towards orchestration-based separation, i.e 3 methods; journeys, davinci, and OIDC (the first two being "embedded" in the existing terminology).

Reply:
Done.